lib/src/audit.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:async';
+
+/// Creates a StreamTransformer which only emits once per [duration],
```

Comment:
A good example might be a text field which auto updates after being modified.  Audit will make more regular updates than debounce, but still always emit the most recent value.

Another way to (mostly) solve the same problem would be to give throttle an optional parameter like `emitLast` which ensures that the last value of the stream is always emitted.